controllers/storagecluster/storagecluster_controller.go

```diff
@@ -155,6 +155,44 @@ func (r *StorageClusterReconciler) SetupWithManager(mgr ctrl.Manager) error {
 		},
 	)
 
+	enqueueStorageProfileRequest := handler.EnqueueRequestsFromMapFunc(
```

Comment:
The name `enqueueStorageProfileRequest` does not describe the fact that we are enqueuing a StorageCluster request. Can we change it to: `enqueueFromStorageProfile`? (storagecluster is implicit as this is the controller for storagecluster CRs)